odl/contrib/datasets/images/examples/cambridge_images.py

```diff
@@ -0,0 +1,45 @@
+"""Example of the Cambridge images."""
+
+import odl.contrib.datasets.images as images
+import numpy as np
+import odl
+
+
+def convert_to_odl(image):
+    """Convert image to ODL object."""
+
+    shape = image.shape
+
+    if len(shape) == 2:
+        space = odl.uniform_discr([0, 0], shape, shape)
+    elif len(shape) == 3:
+        d = shape[2]
+        shape = shape[:2]
+        image = np.transpose(image, (2, 0, 1))
+        space = odl.uniform_discr([0, 0], shape, shape) ** d
+
+    image = space.element(image)
+
+    return image
```

Comment:
I'd expect this to get the image axes a bit wrong. The conversion must be from `ij` to `xy`, which means that

- `shape` must be `reversed(image.shape[:2])`
- the image must (in each channel) be rotated by 90 degrees clockwise by `np.rot90(image, k=-1)`

Reply:
Many of the images get rotated: `im = np.rot90(dct['im'], k=3)` The example file plot all images and they look the way they should do, so I think it is fine the way it is?

Reply:
But then I understand this function wrong. I read this as "take image in Numpy array, return ODL space element", where the array hasn't been manipulated in any way. Expecting an already rotated image is surprising here.

Actually this kind of functionality would be useful in core ODL, too. Converting an image to ODL representation, including rotation and specifying a data type.

Reply:
I don't mind where to do the conversion but I see that this would be very useful for any kind of image and not just here in this example. However, I propose this can be discussed somewhere else and we adjust this example accordingly once we have figured this out.